packages/core/src/domain/configuration/tags.ts

```diff
@@ -0,0 +1,46 @@
+import { display } from '../../tools/display'
+import { InitConfiguration } from './configuration'
+
+export const TAG_SIZE_LIMIT = 200
+
+export function buildTags(configuration: InitConfiguration): string[] {
+  const { env, service, version } = configuration
+  const tags = []
+
+  if (env) {
+    tags.push(buildTag('env', env))
+  }
+  if (service) {
+    tags.push(buildTag('service', service))
+  }
+  if (version) {
+    tags.push(buildTag('version', version))
+  }
+
+  return tags
+}
+
+export function buildTag(name: string, value: string) {
+  value = value.toLowerCase()
+
+  // See https://docs.datadoghq.com/getting_started/tagging/#defining-tags for tags syntax.
+  const valueSizeLimit = TAG_SIZE_LIMIT - name.length - 1
+  if (value.length > TAG_SIZE_LIMIT) {
```

Comment:
> Tags can be up to 200 characters 

should not it be: 
```suggestion
  if (name.length + value.length + 1 > TAG_SIZE_LIMIT) {
```

Reply:
Ah, good catch. I meant to say, `value.length > valueSizeLimit`